utility/utils.py

```diff
@@ -655,7 +656,7 @@ def run_cmd(cmd, **kwargs):
             f"Error during execution of command: {cmd}."
             f"\nError is {r.stderr.decode()}"
         )
-    return r.stdout.decode()
+    return (r.stdout.decode(), r.stderr.decode(), r.returncode)
```

Comment:
I'm not sure I understand this change.
In case of error (i.e. retcode != 0) an exception will be raised and nothing will return
In case of positive flow, you can be sure retcode will be 0 and reterror will be empty

Please elaborate

Reply:
> I'm not sure I understand this change.
> In case of error (i.e. retcode != 0) an exception will be raised and nothing will return
> In case of positive flow, you can be sure retcode will be 0 and reterror will be empty
> 
> Please elaborate

Raz, you are right. My bad, I missed it in my code read that the function is returning the exception when there is an error.

This function seems to be handling well and requires no changes. Apologies from my side.
I'll close this PR.